01_fib/shuutech/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+)
+
+//var out io.Writer = os.Stdout
+var arrayList = []int{0, 1}
+
+func nextVal(a int, b int, n int, c int) []int {
+	numPair := []int{a, b}
+	var i = numPair[0] + numPair[1]
+	var j = numPair[1] + i
+	newPair := []int{i, j}
+	c++
+	if c < n {
+		arrayList = append(arrayList, newPair...)
+		nextVal(i, j, n, c)
+
+	}
+
+	return newPair
+}
+
+func fib(n int) []int {
```

Comment:
due to lab description https://github.com/anz-bank/go-course/tree/master/01_fib function should print and compute fib numbers. In your case logic is printing is done in `main` and `fib` depends on  `nextVal`. Please refactor this.

Reply:
fixed